platforms/documentation/docs/src/docs/userguide/glossary.adoc

```diff
@@ -50,6 +50,65 @@ For more information, see the sections on <<dependency_configurations.adoc#sub:w
 +
 NOTE: The word "configuration" is an overloaded term that has a different meaning outside of dependency management.
 
+[[sub:cross_configuration]]
+Cross-Configuration ::
+See <<sub:cross_project_configuration,Cross-Project Configuration>>.
+
+[[sub:cross_project_configuration]]
+Cross-Project Configuration ::
+Cross-project configuration refers to managing and customizing multiple subprojects within a multi-project build.
++
+It allows you to define common settings, dependencies, and tasks in a shared `build.gradle(.kts)` or `settings.gradle.(kts)` file, typically located in the root project:
++
+[source,groovy]
+.build.gradle
+----
+subprojects {
+    apply plugin: 'java'
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        testImplementation 'junit:junit:4.13.2'
+    }
+}
+----
++
+Cross-project configuration usually breaks `Project Isolation` and `Parallel Projects` so you should use convention plugins or the proper APIs whenever possible:
```

Comment:
```suggestion
Cross-project configuration usually breaks Project Isolation, and Parallel Project Execution so you should use convention plugins or the proper APIs whenever possible:
```

🤔 I don't think PI should be displayed as monospace, it's the name of the feature, not something in code.  I think don't think we should shorthand Parallel Project Execution, at first I thought this was a new feature I had missed.  We should probably link here to a description of PI and to https://docs.gradle.org/current/userguide/performance.html#parallel_execution

Reply:
Fixed.